src/controllers/GenericResponse.js

```diff
@@ -0,0 +1,33 @@
+import Api from '../api/Api';
+
+const lookupUrl = `${process.env.GATEWAY_URL}/lookup`;
+
+const sendAcknowledgement = (res) => {
+  res.send({
+    message: {
+      ack: {
+        status: 'ACK',
+      },
+    },
+  });
+};
+
+const sendErrorWithAuthorization = (res) => {
+  res.status(401).send('Error');
+};
+
+const getPublicKey = async (typeValue) => {
```

Comment:
should we stick to `type` than `typeValue`

Reply:
Done.